journal/updated/02-chapter-1.adoc

```diff
@@ -162,33 +162,36 @@ Leicester, 27th of Sixth month, 1754.
 [.salutation]
 Dear Friend,
 
-I with gladness received your letter, but in the reading of it,
+I with gladness received your letter, but in the reading of it
 was seized with a concern that you are not yet emptied enough of self.
-I hope, my dear friend, you will bear with me, as I must deal honestly;
+I hope, my dear friend, you will bear with me, as I must deal honestly,
 and desire I may speak simply upon all occasions without
 seeking to ornament it with what might please myself,
-because I have learned to know that what is done to gratify self,
+because I have learned to know that what is done to gratify self
 is displeasing to the Almighty.
 It seems to me as if you were in a track that I long trod in,
-therefore I am in much love engaged to tell you a little of what I have experienced.
-When the Lord has been pleased to lay a concern upon me to speak among his people,
+and therefore I am in much love engaged to tell you a little of what I have experienced.
+When the Lord has been pleased to lay a concern upon me to speak among His people,
 I was not willing to stand up or to kneel down, until I had as I thought,
-made it somewhat compact;
+made it somewhat concise;
 and those who were my best friends have been much concerned for me,
-and advised me to give up in faith, for I should never grow while I remained so.
+and advised me to surrender in faith, for I should never grow while I remained so.
 For this cause I have been brought into great confusion,
-so that I could neither go forward nor make an end in a proper manner;
-if I dare have said that I would never preach more, I would have said it;
-but blessed be our God, he is ready to lead us step by step,
-were we but willing to follow him in humility.
-Let all our actions be in his fear, without the least desire for the praise of men;
-then I am satisfied he would forgive all our past offences, and love us freely.
+so that I could neither go forward nor make an end in a proper manner.
+If I dare have said that I would never preach more, I would have said it,
+but blessed be our God, for He is ready to lead us step by step,
+were we but willing to follow Him in humility.
+Let all our actions be in His fear, without the least desire for the praise of men;
+then I am satisfied He would forgive all our past offences, and love us freely.
```

Comment:
the verb tenses here (in the original) are a little weird. I think this sounds more natural: then I am satisfied He **will** forgive